README.md

```diff
@@ -1 +1,288 @@
-# Project-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [ ] Good job adding a documentation :+1: , also good job using the README template provided by microverse, however you should make sure your README stays professional. Kindly make sure that this link is not broken it will make your README file professional :smile:

Reply:
oh i just forgot to remove it

Reply:
i added a LICENSE.md file according to you